src/components/Header/Header.tsx

```diff
@@ -0,0 +1,35 @@
+import { Link } from 'react-router-dom';
+import styles from './Header.module.scss';
+import Logo from '../../../public/images/Logo.png';
+import { BurgerIcon } from '../ui/icons/BurgerIcon';
+import { CartIcon } from '../ui/icons/CartIcon';
+import { FavoriteIcon } from '../ui/icons/FavoriteIcon';
+import { NavBar } from '../NavBar/NavBar';
+
+export const Header: React.FC = () => {
+  return (
+    <>
+      <header className={styles.header}>
+        <Link to="/" className={styles.link}>
+          <img src={Logo} alt="header_logo" className={styles.logo} />
+        </Link>
+
+        <NavBar />
+
+        <div className={styles.header__icons}>
```

Comment:
Change class name so that the class names are in the same style
```suggestion
        <div className={styles.header-icons}>
```